arsenal/modules/config.py

```diff
@@ -22,4 +22,7 @@
 os.environ.setdefault('ESCDELAY', '25')
 os.environ['TERM'] = 'xterm-256color'
 
-savevarfile = join(HOMEPATH, ".arsenal.json")
+if os.environ.get('ARSENAL_LOCAL', 'No') == "No":
```

Comment:
Hey thanks for the idea and the Pull request !
However could you put it this way:
```python
if os.environ.get("ARSENAL_LOCAL"):
  savevarfile = join(os.getcwd(), ".arsenal.json")
else:
  savevarfile = join(HOMEPATH, ".arsenal.json")
```